.pre-commit-config.yaml

```diff
@@ -27,3 +28,26 @@ repos:
     hooks:
       - id: black
         name: Running black in all files.
+
+  - repo: local
```

Comment:
A separate taskipy task like the ones for black and isort could be nice

Reply:
https://github.com/DisnakeDev/disnake/pull/557/commits/03b7ee641befddae754ac11d35c1f4196a33d35d